src/libraries/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpHandler.cs

```diff
@@ -1392,13 +1374,7 @@ private async Task InternalSendRequestBodyAsync(WinHttpRequestState state, bool
         {
             using (var requestStream = new WinHttpRequestStream(state, chunkedModeForSend))
             {
-                await state.RequestMessage.Content.CopyToAsync(
-#if HTTP_DLL
-                    requestStream, state.TransportContext, state.CancellationToken
-#else
-                    requestStream, state.TransportContext
-#endif
-                    ).ConfigureAwait(false);
+                await state.RequestMessage.Content.CopyToAsync(requestStream, state.TransportContext).ConfigureAwait(false);
```

Comment:
If we wanted to we could add a configuration to the System.Net.Http.WinHttpHandler package that would contain a build for .NET 5, and then we could utilize the new APIs in that build.  But maybe it's not worth it.

Reply:
It would only help in scenarios where someone wanted to cancel an operation that uploaded request body content (i.e. POST).

I think we can wait for customer feedback and then decide if it is worth adding that ifdef'd code and additional .NET 5 build for the Nuget package.